metrics_sdk/test/opentelemetry/sdk/metrics/instrument/histogram_test.rb

```diff
@@ -35,4 +35,29 @@
     _(last_snapshot[0].data_points[0].attributes).must_equal('foo' => 'bar')
     _(last_snapshot[0].aggregation_temporality).must_equal(:delta)
   end
+
+  it 'can apply advice to change the explicit bucket boundaries' do
+    histogram = meter.create_histogram('histogram', unit: 's', description: 'test', advice: { explicit_bucket_boundaries: [5, 10, 15] })
+
+    histogram.record(0)
+    histogram.record(5)
+    histogram.record(5)
+    histogram.record(10)
+    histogram.record(20)
+
+    metric_exporter.pull
+    last_snapshot = metric_exporter.metric_snapshots.last
+    _(last_snapshot[0].data_points[0].bucket_counts).must_equal([3, 1, 0, 1])
+
+    _(last_snapshot[0].name).must_equal('histogram')
+    _(last_snapshot[0].unit).must_equal('s')
+    _(last_snapshot[0].description).must_equal('test')
+    _(last_snapshot[0].instrumentation_scope.name).must_equal('test')
+    _(last_snapshot[0].data_points[0].count).must_equal(5)
+    _(last_snapshot[0].data_points[0].sum).must_equal(40)
+    _(last_snapshot[0].data_points[0].min).must_equal(0)
+    _(last_snapshot[0].data_points[0].max).must_equal(20)
+    _(last_snapshot[0].data_points[0].attributes).must_be_nil
+    _(last_snapshot[0].aggregation_temporality).must_equal(:delta)
```

Comment:
idk if we need all these assertions, when really we just want to make sure the bucket was assigned properly. it felt safer JIC adding the new bucket boundaries somehow interfered with the bucket-related calculations (ex. `sum`)